tests/conftest.py

```diff
@@ -81,6 +84,9 @@ def verify_cleanup():
             inst.stop()
         pytest.exit(f"Detected non stopped instances ({count}), aborting test run")
 
+    threads = frozenset(threading.enumerate()) - threads_before
+    assert not threads
```

Comment:
Could we print the thread names?

Reply:
It should. Pytest should expand the assert. But I don't remember if it actually did. You need to look at where the error is generated I think. Not at the summary.

Reply:
```
E       assert not frozenset({<Thread(SyncWorker_1, started daemon 140181915428608)>, <Thread(SyncWorker_2, started daemon 140181932214016)>})
```

You're right it does